pkg/kds/zone/components.go

```diff
@@ -24,17 +24,19 @@ import (
 )
 
 var (
-	kdsZoneLog    = core.Log.WithName("kds-zone")
+	kdsZoneLog = core.Log.WithName("kds-zone")
+
 	ProvidedTypes = []model.ResourceType{
-		mesh.ZoneIngressType,
-		mesh.ZoneIngressInsightType,
-		mesh.DataplaneType,
 		mesh.DataplaneInsightType,
+		mesh.DataplaneType,
+		mesh.ZoneIngressInsightType,
+		mesh.ZoneIngressType,
 	}
+
+	// ConsumedTypes is the list of Kuma resource types that a KDS instance will request from its peer.
```

Comment:
It's not really accurate. Today we have 5 lists of resources (and hopefully we will refactor this soon):

pkg/kds/zone:
- ConsumedTypes (types consumed by Zone CP from Global CP)
- ProvidedTypes (types provided by Zone CP to Global CP)

pkg/kds/global:
- ConsumedTypes (types consumed by Global CP from Zone CP)
- ProvidedTypes (types provided by Global CP to Zone CP)

pkg/kds:
- SupportedTypes (all types used in KDS)

So this comment should sound something like this:
```go
// ConsumedTypes is the list of Kuma resource types that Zone CP instance will request from Global CP over KDS
```

Reply:
Thanks! I updated all the comments based on your list above.